front/app/components/ProjectCard/LargeProjectCard.tsx

```diff
@@ -0,0 +1,333 @@
+  showFollowButton?: boolean;
+}
+
+const LargeProjectCard = memo<InputProps>(
+  ({ projectId, hideDescriptionPreview, className, showFollowButton }) => {
+    const { data: project } = useProjectById(projectId);
+    const { data: projectImage } = useProjectImage({
+      projectId,
+      imageId: project?.data.relationships.project_images?.data[0]?.id,
+    });
+    const currentPhaseId =
+      project?.data?.relationships?.current_phase?.data?.id ?? null;
+    const { data: phase } = usePhase(currentPhaseId);
+    const fetchPhases = project && !currentPhaseId;
+    const { data: phases } = usePhases(
+      fetchPhases ? project.data.id : undefined
+    );
+    const theme = useTheme();
+
+    if (project) {
+      const imageUrl = !projectImage
+        ? null
+        : projectImage.data.attributes.versions?.large;
+      const projectUrl: RouteType = getProjectUrl(project.data);
+      const isArchived =
+        project.data.attributes.publication_status === 'archived';
+      const hasAvatars =
+        project.data.relationships.avatars &&
+        project.data.relationships.avatars.data &&
+        project.data.relationships.avatars.data.length > 0;
+      const showFooter = hasAvatars;
+      const avatarIds =
+        project.data.relationships.avatars &&
+        project.data.relationships.avatars.data
+          ? project.data.relationships.avatars.data.map((avatar) => avatar.id)
+          : [];
+      const inputTerm = getInputTerm(phases?.data, phase?.data);
+
+      return (
+        <Container
+          className={[
+            className || '',
+            'e2e-project-card',
+            'e2e-admin-publication-card',
+            isArchived ? 'archived' : '',
+            hideDescriptionPreview ? 'hideDescriptionPreview' : '',
+          ]
+            .filter((item) => item)
+            .join(' ')}
+          to={projectUrl}
+          scrollToTop
+          onClick={() => {
+            handleProjectCardOnClick(project.data.id);
+          }}
+        >
+          <ScreenReaderContent project={project} />
+
+          <ProjectImageContainer>
+            {imageUrl ? (
+              <ProjectImage src={imageUrl} alt="" cover={true} />
+            ) : (
+              <Box
+                display="flex"
+                alignItems="center"
+                justifyContent="center"
+                flex="1"
+                background={colors.grey300}
+              >
+                <Icon
+                  name="building"
+                  width="80px"
+                  height="80px"
+                  fill={colors.white}
+                />
+              </Box>
+            )}
+          </ProjectImageContainer>
+
+          <ProjectContent>
+            <ProjectHeader
+              phase={phase}
+              inputTerm={inputTerm}
+              project={project}
+              ContentHeaderComponent={ContentHeader}
+            />
+            <ContentBody aria-hidden>
+              <ProjectTitle
```

Comment:
Same for this part (259-286)- corresponds to 254-281 in the medium card and 267-294 in the small card